File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.js
function solution(arr1, arr2) {
    var ans = [];
    for (let i = 0; i < arr1.length; i++) {
        var list = [];
        for (let j = 0; j < arr1[i].length; j++) {
            list.push(arr1[i][j] + arr2[i][j])
        }
        ans.push(list);
    }
    return ans;
}

File: 프로그래머스/1/12950. 행렬의 덧셈/행렬의 덧셈.py
def solution(arr1, arr2):
    c_l = len(arr1)  # 열 길이
    r_l = len(arr1[0])  # 행 길이
    ans = [[0] * r_l for _ in range(c_l)]
    
    for i in range(c_l):
        for j in range(r_l):
            ans[i][j] = arr1[i][j] + arr2[i][j]
    
    return ans